exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,21 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
```

Comment:
The function is correct. Could you take a look if you could sort it out in order when validate it?

for example: line 16: instead of that line, write 60 > mark && mark < 80. You would read from left to write.